service/history/api/workflow_lease.go

```diff
@@ -44,6 +44,7 @@ type workflowLease struct {
 type UpdateWorkflowAction struct {
 	Noop               bool
 	CreateWorkflowTask bool
+	AbortUpdates       bool
```

Comment:
it feels like a different type of post actions compared to the other two... better to add some comments here...